src/options.ts

```diff
@@ -36,7 +36,7 @@ export type Options = {
   stringEnums: boolean;
   constEnums: boolean;
   outputClientImpl: boolean | 'grpc-web';
-  outputServices: false | 'grpc-js';
+  outputServices: boolean | 'grpc-js';
```

Comment:
Not sure whether it should be a separate option. Perhaps someone might want to have both `grpc-js` and generic definitions

Reply:
How about `outputServices: false | 'grpc-js' | 'generic-definitions` and then maybe someday (doesn't have to be now) we could support like csv of `outputServices=grpc-js,generic-definitions`?

Reply:
Done